Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,90 @@
-import React from 'react';
+import React, {Component,} from 'react';
 import './App.css';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import ButtonGroup from './components/ButtonGroup/ButtonGroup';
+import NumberGroup from './components/ButtonGroup/NumberGroup';
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
+class App extends Component {
```

Comment:
Try to use your App.js as a presentational container and put the logic in the component that needs it.